hw/ip/clkmgr/doc/checklist.md

```diff
@@ -0,0 +1,232 @@
+---
+title: "CLKMGR Checklist"
+---
+
+<!--
+NOTE: This is a template checklist document that is required to be copied over to the 'doc'
+directory for a new design that transitions from L0 (Specification) to L1 (Development)
+stage, and updated as needed. Once done, please remove this comment before checking it in.
+-->
+This checklist is for [Hardware Stage]({{< relref "/doc/project/development_stages.md" >}}) transitions for the [CLKMGR peripheral.]({{< relref "hw/ip/clkmgr/doc" >}})
+All checklist items refer to the content in the [Checklist.]({{< relref "/doc/project/checklist.md" >}})
+
+## Design Checklist
+
+### D1
+
+Type          | Item                           | Resolution  | Note/Collaterals
+--------------|--------------------------------|-------------|------------------
+Documentation | [SPEC_COMPLETE][]              | WIP         | [CLKMGR Design Spec]({{<relref "hw/ip/clkmgr/doc" >}})
+Documentation | [CSR_DEFINED][]                | Done        |
+RTL           | [CLKRST_CONNECTED][]           | Done        |
+RTL           | [IP_TOP][]                     | Done        |
+RTL           | [IP_INSTANTIABLE][]            | Done        |
+RTL           | [PHYSICAL_MACROS_DEFINED_80][] | NA          |
+RTL           | [FUNC_IMPLEMENTED][]           | Done        |
+RTL           | [ASSERT_KNOWN_ADDED][]         | Not Started |
```

Comment:
it looks like this one is done as well?
https://github.com/lowRISC/opentitan/blob/ae7e2ef813774a3bf7e4cd9cea5b0eb1cdf3e218/hw/ip/rstmgr/rtl/rstmgr.sv#L534-L544

Reply:
fixed.